content/articles/two-factor-authentication-enforcement.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: Enforce Two-Factor Authentication for a shared Account
+excerpt: How to protect your shared DNSimple account using Two-Factor authentication enforcement.
+categories:
+- Account
+---
+
+# Enforce Two-Factor Authentication for a shared Account
+
+<info>
+This article describes a feature in Public Beta.
+</info>
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+DNSimple supports [Two-Factor authentication](/articles/two-factor-authentication/). This feature enforces that all users in our account must have Two-Factor authentication enabled.
+
+When you have added other users to your account with it is important to ensure the security of each account member. This is why this feature is available to all plans that have access to the teams feature.
```

Comment:
This sentence is quite confusing. I can't understand what it means. Also, please let's not refer as the Team feature.